sample_code/ep009/README.md

```diff
@@ -21,7 +21,7 @@ pip install future-breakpoint
 
 ### Python debugger (pdb)
 
-```python
+```console
```

Comment:
`console` is supposed to be a shell console, I would probably drop the language here since there isn't one that really represents this well